runner/utils/driver/istio/logs.go

```diff
@@ -0,0 +1,59 @@
+package istio
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"k8s.io/api/core/v1"
+
+	"github.com/spaceuptech/space-cloud/runner/utils"
+)
+
+// GetLogs get logs of specified services
+func (i *Istio) GetLogs(ctx context.Context, isFollow bool, projectID, taskID, replica string) (io.ReadCloser, error) {
+	if taskID == "" {
+		arr := strings.Split(replica, "--")
+		if len(arr) < 2 {
+			return nil, utils.LogError("Invalid replica id", "docker", "get-logs", nil)
+		}
+		taskID = arr[0]
```

Comment:
This seems to be wrong. Did you test it?

Reply:
yes